added documentation for new callback method of creating ext (#176)
ueberdosis/tiptap-docs: src/content/editor/extensions/custom-extensions/create-new/extension.mdx

})
```

You can also use a callback function to create an extension. This is useful if you want to encapsulate the logic of your extension, for example when you want to define event handlers or other custom logic.

```js
import { Extension } from '@tiptap/core'

const CustomExtension = Extension.create(() => {

>   // here you could define variables or function that you can use on your schema definition

Reviewer: ```suggestion
  // Define variables or functions to use inside your extension
```

Reviewer: Simplified the comment
